### PR TITLE
Setup GitHub Actions and Dependabot updates

### PR DESCRIPTION
**Pull request by ryanwi**

### CODE REVIEW

**Comment by lpradovera** [comment]
This is just a maintenance pass, correct?

**Comment by ryanwi** [comment]
yes, a the foundation/prep to evaluate further updates.

**Comment by lpradovera** [comment]
We should probably add Ruby 3.2 to the CI list, as it is the latest 3 version.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    branches: [ "master" ]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: ['2.6', '2.7', '3.0']
```
Comment (ryanwi) [inline .github/workflows/ci.yml]:
```suggestion
        ruby-version: ['2.6', '2.7', '3.0', '3.1', '3.2']
```

**Review by lpradovera — APPROVED** [review]
🚢

**Pull request merged**